clusters/app.ci/registry-access/rhcos-devel/admin_manifest.yaml

```diff
@@ -0,0 +1,55 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  annotations:
+    openshift.io/description: Automation ServiceAccounts for RHCOS Devel
+    openshift.io/display-name: RHCOS Devel
+  name: rhcos-devel
+---
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: image-pusher
+  namespace: rhcos-devel
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: rhcos-devel-image-pusher-binding
+  namespace: rhcos-devel
+roleRef:
+  kind: ClusterRole
+  apiGroup: rbac.authorization.k8s.io
+  name: system:image-builder
+subjects:
+  - kind: ServiceAccount
+    namespace: rhcos-devel
+    name: image-pusher
+---
+kind: Group
+apiVersion: v1
+metadata:
+  name: rhcos-devel-admins
+users:
+  - ashcrow
+  - cgwalters
+  - dustymabe
+  - jlebon
+  - miabbott
+  - mike-nguyen
+---
+kind: RoleBinding
```

Comment:
Can you also add this so you can see stuff in the console and use `oc` (with your values substituted):

```
---
# this adds the admins to the project.
kind: RoleBinding
apiVersion: rbac.authorization.k8s.io/v1
metadata:
  name: my-project-viewer-binding
  namespace: my-project
roleRef:
  kind: ClusterRole
  apiGroup: rbac.authorization.k8s.io
  name: view
subjects:
  - kind: Group
    apiGroup: rbac.authorization.k8s.io
    name: my-project-admins
    namespace: my-project
```